cluster/node-pools/master-default/userdata.yaml

```diff
@@ -140,6 +140,7 @@ write_files:
           - --service-account-signing-key-file=/etc/kubernetes/ssl/service-account-private-key.pem
           - --service-account-issuer={{ .Cluster.APIServerURL }}
           - --service-account-jwks-uri={{ .Cluster.APIServerURL }}/openid/v1/jwks
+          - --service-account-extend-token-expiration={{ .Cluster.ConfigItems.rotate_service_account_tokens_extended_expiration }}
```

Comment:
The default is `true` already but this way we have to explicitly disable it or remove the flag.